src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/operations/live_event.py

```diff
@@ -55,7 +55,7 @@ def create_live_event_preview(preview_locator, streaming_policy_name,
     from azure.mgmt.media.models import (IPAccessControl, LiveEventPreviewAccessControl, LiveEventPreview)
 
     allow_list = []
-    if preview_ips is None:
+    if preview_ips is None or preview_ips[0] == 'AllowAll':
```

Comment:
preview_ips is **not** None **and**
We will leave it empty if user don't set it